README.md

```diff
@@ -121,7 +120,7 @@ var authClient = new OktaAuth({url: 'https://acme.okta.com'});
 
 ## signIn(options)
 
-The goal of an authentication flow is to set an Okta session cookie on the user's browser. The flow is started using `signIn`.
+The goal of an authentication flow is to set an Okta session cookie on the user's browser or exchange for an `id_token` or `access_token`. The flow is started using `signIn`.
```

Comment:
or **retrieve** an id_token or access_token (or any other word that's better than retrieve, like get, etc)